app/_how-tos/kic-get-started-proxy-caching.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Proxy Caching
+description: |
+  Cache all GET and HEAD responses across all Services using Proxy Cache and the KongClusterPlugin resource
+content_type: how_to
+
+permalink: /kubernetes-ingress-controller/get-started/proxy-caching/
+breadcrumbs:
+  - /kubernetes-ingress-controller/
+  - index: kubernetes-ingress-controller
+    section: Get Started
+
+series:
+  id: kic-get-started
+  position: 4
+
+tldr:
+  q: How do I cache upstream responses using {{ site.kic_product_name }}?
+  a: |
+    Use the `proxy-cache` plugin by creating a `KongPlugin` resource while specifying `config.response_code`, `config.request_method` and `config.cache_ttl`.
+
+products:
+  - kic
+
+tools:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+
+prereqs:
+  skip_product: true
+---
+
+## About the Proxy Cache plugin
+
+One of the ways {{site.base_gateway}} delivers performance is through caching. The [Proxy Cache plugin](/plugins/proxy-cache/) accelerates performance by caching responses based on configurable response codes, content types, and request methods. When caching is enabled, upstream services are not impacted by repetitive requests, because {{site.base_gateway}} responds on their behalf with cached results. Caching can be enabled on specific Routes or for all requests globally.
+
+## Proxy Cache headers
+The `proxy-cache` plugin returns a `X-Cache-Status` header that can contain the following cache results:
+
+{% include_cached /plugins/caching/cache-header.md %}
+
+## Create a proxy-cache KongClusterPlugin
+
+In the previous section you created a `KongPlugin` that was applied to a specific service or route. You can also use a `KongClusterPlugin` which is a global plugin that applies to all services.
+
+This configuration caches all `HTTP 200` responses to `GET` and `HEAD` requests for 300 seconds:
+
+{% entity_example %}
+type: plugin
+data:
+  name: proxy-cache-all-endpoints
+  plugin: proxy-cache
+  config:
+    response_code:
+    - 200
+    request_method:
+    - GET
+    - HEAD
+    content_type:
+    - text/plain; charset=utf-8
+    cache_ttl: 300
+    strategy: memory
+{% endentity_example %}
+
+## Test the proxy-cache plugin
+
+To test the proxy-cache plugin, send another six requests to `$PROXY_IP/echo`:
+
+{% validation rate-limit-check %}
+iterations: 6
+url: '/echo'
+headers:
+  - 'apikey:example-key'
+on_prem_url: $PROXY_IP
+konnect_url: $PROXY_IP
+grep: "(Status|< HTTP)"
+message: null
+output:
+  explanation: |
+    The first request results in `X-Cache-Status: Miss`. This means that the request is sent to the upstream service. The next four responses return `X-Cache-Status: Hit` which indicates that the request was served from a cache.
```

Comment:
It's very easy to do this part of the guide too quickly after the Rate Limiting portion and get rate limited, so you don't see any proxy caching info.

Reply:
I hit this too, but wondered if it was just that we're flying through when testing.

Would you add a note here saying to wait 60s?

Reply:
You could also decrease the rate limiting window to something like 30 seconds. Doesn't hurt to add a note just in case.

Reply:
Decreasing the rate limiting window makes a lot more sense. Given we use a loop, 10 seconds should be enough

Reply:
Ah, we'd have to use RLA to have that level of granularity. We can do per second or per minute in RL